lang/cs/BUILD.md

```diff
@@ -24,7 +24,13 @@ Prerequisites
 -------------
 
   * Windows OS.
-  * [Building REEF Java](../java/BUILD.md).
+  * Apache maven
```

Comment:
@tcNickolas The issue @skanjila and @EstusDev faced was that the .NET build instructions weren't self contained and it wasn't obvious what subset of the Java build they needed to do to get the .NET build going. The text above is taken from my notes on a recent test of building REEF.NET on a fresh VM.